app/views/users/_admins.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2 class="section-label">Admins</h2>
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% admins.each do |admin| %>
+      <tr>
+        <td><%= admin.name %></td>
+        <td class="table-actions"><a href="" class="table-link table-delete">Delete</a></td>
```

Comment:
I know you're moving this around right now, but it might be a good time to add your translations